catkit/emulators/iris_ao_controller.py

```diff
@@ -48,7 +48,7 @@ def convert_command_to_poppy_surface(dm_command_dict):
 
         # Setting the simulated IrisAO means setting each actuator individually
         for seg, values in convert_command_to_poppy_surface(new_surface).items():
-            self.set_actuator(seg-1, values[0] * u.um, values[1] * u.mrad, values[2] * u.mrad)  # TODO: double-check the -1 here, meant to correct for different segment names
+            self.set_actuator(seg-1, values[0] * u.um, values[2] * u.mrad, values[1] * u.mrad)  # TODO: double-check the -1 here, meant to correct for different segment names
```

Comment:
Instead of _just_ a comment we can expand the code a little also, i.e.,
```python
piston_stroke = values[0] * um
tip = values[1] * umrad
tilt = values[2] * umrad
self.set_actuator(seg-1, piston_stroke, tip, tilt)
```
(correcting for which ever way around it actually is 🤷 )

Reply:
I like this

Reply:
We then only need to comment about what expects what in which order, rather than also commenting on index semantics.